plugins/source/azure/resources/services/redis/redis.go

```diff
@@ -1,278 +0,0 @@
-package redis
```

Comment:
Renamed to `plugins/source/azure/resources/services/redis/caches.go` as that's the correct term in Azure